torch/__init__.py

```diff
@@ -867,6 +867,7 @@ def _assert(condition, message):
 import torch.backends.mkldnn
 import torch.backends.openmp
 import torch.backends.quantized
+from torch.backends import xpu as _xpu
```

Comment:
Do you want `_xpu` to be private?

If you do, then for consistency, can you rename the `torch/xpu` folder to `torch/_xpu`? Otherwise we're in an inconsistent state, where a user can use `torch._xpu` (appears private), but also `import torch.backends.xpu` (appears public)

Reply:
Also - `torch._xpu` is probably not what we want - all other backend init files show up as `torch.backends.<foo>`.